src/components/table/OcTableFiles.vue

```diff
@@ -382,7 +382,7 @@ export default {
   methods: {
     fileDragged(file) {
       const selectedResourceInResources = this.selectedResources.some(e => e.id === file.id)
-      if (selectedResourceInResources) {
+      if (!selectedResourceInResources) {
```

Comment:
Just gave it a try (by building it locally and linking it to https://github.com/owncloud/web/pull/5588) and I don't think that's the right way to go, at least for single file drag&drop: As soon as you start dragging your single file of choice, the `Move` and `Copy` buttons appear above the table (since, once you start dragging, you now have a selected resource) causing the UI to become "jumpy"

Reply:
Could we discuss a solution if you have time? I'm not really sure what we can do about this since we need the selection in order to move files.

Reply:
Figured out that this "jumpy" behaviour will get fixed by moving the bulk actions into the right sidebar so good for now